doc/release/release_0.20.rst

```diff
@@ -0,0 +1,479 @@
+- Find peaks at border with `peak_local_max with `exclude_border=0` (`#6502 <https://github.com/scikit-image/scikit-image/pull/6502>`_)
+- Fix resize anti_aliazing default value when input dtype is integer and order == 0 (`#6503 <https://github.com/scikit-image/scikit-image/pull/6503>`_)
+- Add Github actions/stale to label "dormant" issues and PRs (`#6506 <https://github.com/scikit-image/scikit-image/pull/6506>`_)
+- Clarify header pointing to notes for latest version released. (`#6508 <https://github.com/scikit-image/scikit-image/pull/6508>`_)
+- Reduce ridge filters memory footprints (`#6509 <https://github.com/scikit-image/scikit-image/pull/6509>`_)
+- Update benchmark environment to recent Python and NumPy versions (`#6511 <https://github.com/scikit-image/scikit-image/pull/6511>`_)
+- Add new flag to convex_hull_image and grid_points_in_poly (`#6515 <https://github.com/scikit-image/scikit-image/pull/6515>`_)
+- relax label name comparison in benchmarks.yaml (`#6520 <https://github.com/scikit-image/scikit-image/pull/6520>`_)
+- update plot_euler_number.py for maplotlib 3.6 compatibility (`#6522 <https://github.com/scikit-image/scikit-image/pull/6522>`_)
+- Use mask during rescaling in segmentation.slic and improve handling of error cases (`#6525 <https://github.com/scikit-image/scikit-image/pull/6525>`_)
+- make non-functional change to build.txt to fix cache issue on CircleCI (`#6528 <https://github.com/scikit-image/scikit-image/pull/6528>`_)
+- update setup.cfg field from license_file to license_files (`#6529 <https://github.com/scikit-image/scikit-image/pull/6529>`_)
+- Fix wrong doc on connected pixels in flood (`#6534 <https://github.com/scikit-image/scikit-image/pull/6534>`_)
+- Minor doc fix: add missing print statement in the `plot_segmentations.py` example (`#6535 <https://github.com/scikit-image/scikit-image/pull/6535>`_)
+- Apply codespell to fix common spelling mistakes (`#6537 <https://github.com/scikit-image/scikit-image/pull/6537>`_)
+- Ignore codespell fixes with git blame (`#6539 <https://github.com/scikit-image/scikit-image/pull/6539>`_)
+- Add missing spaces to regionprops error message. (`#6545 <https://github.com/scikit-image/scikit-image/pull/6545>`_)
+- Update "Mark dormant issues" workflow (`#6546 <https://github.com/scikit-image/scikit-image/pull/6546>`_)
+- Add missing space in math directive in normalized_mutual_information's docstring (`#6549 <https://github.com/scikit-image/scikit-image/pull/6549>`_)
+- Add missing option stale-pr-label for "Mark dormant issues" workflow (`#6552 <https://github.com/scikit-image/scikit-image/pull/6552>`_)
+- Remove FUNDING.yml in preference of org version (`#6553 <https://github.com/scikit-image/scikit-image/pull/6553>`_)
+
+Pull Requests in this release
+-----------------------------
+- Add benchmarks for morphology.local_maxima (`#3255 <https://github.com/scikit-image/scikit-image/pull/3255>`_)
```

Comment:
We should make sure there are no duplicates

Reply:
I intend to fully delete the previous section "Todo". Whether we want to include this short list of all PR at the end similar to the contributor and reviewer lists is up to you.